modin/pandas/test/utils.py

```diff
@@ -203,6 +204,16 @@
 test_data_categorical_values = list(test_data_categorical.values())
 test_data_categorical_keys = list(test_data_categorical.keys())
 
+# Fully fill all of the partitions used in tests.
+test_data_large_categorical_dataframe = {
+    i: pandas.Categorical([0] * (NPartitions.get() * 32))
```

Comment:
note that this does not represent the original reproducer - in the #1628 all the category entries were distinct while here you create a "NPartitions * 32" entries of a single category

Reply:
Fixed.